src/components/Table/sortRows.ts

```diff
@@ -68,7 +70,9 @@ function sortValue(value: ReactNode | GridCellContent): any {
   if (maybeFn instanceof Function) {
     return maybeFn();
```

Comment:
Do we need to lower case here as well? Maybe not early return an do `maybeFn = maybeFn()`?

Reply:
Makes sense. Thanks!